### PR TITLE
Form validation not running in join section on refreshing the window

### PR DESCRIPTION
**Pull request by fakhruddinkw**
### What is the change?
Added a window.onload, so that whenever, a user hits refresh button, we can run the validation of the form again.

Before :
https://user-images.githubusercontent.com/85249044/235845751-a3a034d6-9ffc-4282-ab20-a41b39879b2e.mp4

After :
https://user-images.githubusercontent.com/85249044/235845332-d27d6e63-54e5-4818-962e-3c60afba85a2.mp4

### CODE REVIEW

File: js/join.js [hunk 1]
```diff
@@ -6,6 +6,26 @@ import {
   SELF_URL,
 } from './constants.js';
 
+const reloadFunc = () => {
+  let flowState = +window.localStorage.getItem('flowState');
+  flowState += 1;
+  const currentPage = document.getElementById('page' + flowState);
+  const inputElements = currentPage.getElementsByTagName('input');
```
Comment (shubhamsigdar1) [inline js/join.js:13]: You are selecting all the input of particular page...Is I am right??

**Review by shubhamsigdar1 — APPROVED** [review]
LGTM

File: js/join.js (hunk 1 above)
Comment (fakhruddinkw) [reply js/join.js:13]: Yes

**Review by bhtibrewal — APPROVED** [review]
looks okay to me

**Review by RitikJaiswal75 — APPROVED** [review]

**Pull request merged**